tabbycat/privateurls/utils.py

```diff
@@ -55,9 +55,7 @@ def send_randomised_url_emails(request, tournament, queryset, url_name, url_key_
         url = request.build_absolute_uri(path)
 
         formatted_subject = subject
-        formatted_message = message.replace('%name', instance.name).replace('%url', url)
-        if hasattr(instance, 'team'):
-            formatted_message = formatted_message.replace('%team', instance.team.short_name)
+        formatted_message = message.replace('<NAME>', instance.name).replace('<URL>', url).replace('<TEAM>', instance.team.short_name)
```

Comment:
This crashes with adjudicators: the `instance.team.short_name` isn't short-circuited when `'<TEAM>'` isn't found, so with adjudicators it raises an `AttributeError`. The old `hasattr()` check is good.